### PR TITLE
SPLAT-789: Periodic tests for provider cert tool on non integrated in stall (platform: none)

### PR DESCRIPTION
**Pull request by bostrt**
The provider certification tool needs to be tested with non-integrated (platform: none) installations of OpenShift more than IPI at this point. I've added the periodic test and adjusted the tests to run:
- Platform None Test - 3 days a week
- vSphere IPI Test - 2 days a week

### CODE REVIEW

**Comment by bostrt** [comment]
Bootstrap is failing to pull release image. I must be missing some secret:

```
Sep 28 20:32:31 bootstrap-0 release-image-download.sh[1803]: Pull failed. Retrying registry.apps.build01-us-west-2.vmc.ci.openshift.org/ci-op-i0f09ql1/release@sha256:fe4d499ac9fc7d12fcfccf3d6ae8a916c31e282d18adbebb0456c0fd6aef02c9...
Sep 28 20:32:31 bootstrap-0 release-image-download.sh[53555]: Error: initializing source docker://registry.apps.build01-us-west-2.vmc.ci.openshift.org/ci-op-i0f09ql1/release@sha256:fe4d499ac9fc7d12fcfccf3d6ae8a916c31e282d18adbebb0456c0fd6aef02c9: reading manifest sha256:fe4d499ac9fc7d12fcfccf3d6ae8a916c31e282d18adbebb0456c0fd6aef02c9 in registry.apps.build01-us-west-2.vmc.ci.openshift.org/ci-op-i0f09ql1/release: unauthorized: authentication required
```

**Comment by bostrt** [comment]
>  Error: error fetching virtual machine: ServerFaultCode: The object 'vim.Folder:group-v377258' has already been deleted or has not been completely created 

This issue comes up occasionally and we believe its a bug in vsphere.

/test pj-rehearse

File: ci-operator/step-registry/provider-certification-tool/run/provider-certification-tool-run-commands.sh [hunk 1]
```diff
@@ -6,8 +6,16 @@ set -o pipefail
 
 export KUBECONFIG=${SHARED_DIR}/kubeconfig
 
-# Run the tool in dedicated mode with watch flag set.
-./openshift-provider-cert-linux-amd64 run --watch --dedicated > /dev/null
+count=$(oc get nodes -l node-role.kubernetes.io/tests= 2>/dev/null | wc -l)
```
Comment (bostrt) [inline ci-operator/step-registry/provider-certification-tool/run/provider-certification-tool-run-commands.sh:9]: failing with 'command not found' because bin image is being used.

**Comment by bostrt** [comment]
Timeout during UPI install, trying this again:

/test pj-rehearse

**Comment by mtulio** [comment]
@bostrt  💯  LGTM
I would ask @rvanderp3 to take a look too.

File: ci-operator/step-registry/provider-certification-tool/run/provider-certification-tool-run-commands.sh [hunk 2]
```diff
@@ -4,10 +4,15 @@ set -o nounset
...
 set -o pipefail
 
-export KUBECONFIG=${SHARED_DIR}/kubeconfig
-
-# Run the tool in dedicated mode with watch flag set.
-./openshift-provider-cert-linux-amd64 run --watch --dedicated > /dev/null
+if [ -f "${SHARED_DIR}/dedicated" ]
+then
+  # Run the tool in dedicated mode with watch flag set.
+  echo "Found node dedicatd to provider tool"
```
Comment (rvanderp3) [inline ci-operator/step-registry/provider-certification-tool/run/provider-certification-tool-run-commands.sh:10]:
```suggestion
  echo "Found node dedicated to provider tool"
```
